### ISSUE TITLE
When defining new ingestion via the UI, API call fails with 400 and UI gives no warning

### ISSUE THREAD

**Issue by juliarvalenti** [open-metadata/OpenMetadata#2289]
https://user-images.githubusercontent.com/42276144/150172068-29aea00f-b5d8-4e57-987d-0a2eb9e71909.mov

As in attached video - when creating a new ingestion via the UI, a 400 error is received.
`{"code":400,"message":"[id must not be null]"}`

The UI gives no warning about this and just displays an empty list.

The POST request is: 

```
{"ingestionType":"mysql","displayName":"my_new_ingest","name":"my_new_ingest","service":{"id":"ea51833f-3cf9-44b6-a82c-f3f066ed422b","type":"databaseService","name":"airport_db"},"startDate":"2022-01-19","endDate":"","scheduleInterval":"5 * * * *","forceDeploy":true,"connectorConfig":{"database":"airport","enableDataProfiler":false,"excludeFilterPattern":["mysql, information_schema, performance_schema, sys"],"host":"my-host.us-west-1.rds.amazonaws.com","includeFilterPattern":[],"includeViews":true,"password":"demo_user","username":"demo_pass"},"owner":{"type":"user"}}
```

The CURL is 

```
curl 'http://localhost:8585/api/operations/v1/ingestion' -X POST -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:96.0) Gecko/20100101 Firefox/96.0' -H 'Accept: application/json, text/plain, */*' -H 'Accept-Language: en-US,en;q=0.5' -H 'Accept-Encoding: gzip, deflate' -H 'Content-Type: application/json;charset=utf-8' -H 'Origin: http://localhost:8585' -H 'Connection: keep-alive' -H 'Referer: http://localhost:8585/ingestion' -H 'Cookie: bid=6724d19e-c9ff-4026-bb53-0e1dc5b58d54; session=.eJw1kM1OxDAMhN8l55Wa_8R9FYQq17G3K0q7StIDQrw7AcTRnplPo_lUi1Rum5p7vfimlkdRsxJCH4wVq40OiMUx5qydd1DE6SjW-xU9CQk6nTCbn49g1slASiGMAIChAE5ntzqNNATwhTIQZgjWcdIYKZaYxLoogquHQDat0YdU1Cjy5PqOBx_9vxq1Kks_3_gYDRNYi1CQxUQmguwSB2uNzYDRleIDkR_kQSp4X1rHfrVFHnvnOuK470PZT8KdxzmQN_XEOy_bo_Wzfqj5RW29P-dp-jVtZ-tz1llPg1avY9qHb1KvN3U1rn-jGfX1Dda3ZOU.Yeg5RQ.LfP69B683F4HUyg3PuESvfEQ71Y; VERSION_0_6_0=true' -H 'Sec-Fetch-Dest: empty' -H 'Sec-Fetch-Mode: cors' -H 'Sec-Fetch-Site: same-origin' --data-raw '{"ingestionType":"mysql","displayName":"my_new_ingest","name":"my_new_ingest","service":{"id":"ea51833f-3cf9-44b6-a82c-f3f066ed422b","type":"databaseService","name":"airport_db"},"startDate":"2022-01-19","endDate":"","scheduleInterval":"5 * * * *","forceDeploy":true,"connectorConfig":{"database":"airport","enableDataProfiler":false,"excludeFilterPattern":["mysql, information_schema, performance_schema, sys"],"host":"my-host.us-west-1.rds.amazonaws.com","includeFilterPattern":[],"includeViews":true,"password":"demo_user","username":"demo_pass"},"owner":{"type":"user"}}'
```

**Comment by juliarvalenti** [comment]
I am able to get this to work now, and I can no longer reproduce the bug.  I am not sure what I changed though!

**Comment by szecsipeter2** [comment]
I have exactly same error when I would like to add ingestion to my postgres database service.
I used https://github.com/open-metadata/openmetadata-helm-charts for installation.
What should I modify to be able to add ingestion?

Ps: I checked out openmetadata-dependencies-0.0.6 at https://github.com/open-metadata/openmetadata-helm-charts.

**Comment by sbbagal13** [comment]
Is this really solved ? I am facing same issue with 0.8.0 setup with external ES, Mysql and Airflow 
Not getting exact error messages where its failing and why
